File: Daily Test/C/Toggle Vowels - Both Ends 3.3.2026.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int isVowel(char ch) {
    return ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u';
}

int main() {
    char s[1000];
    scanf("%s", s);

    while(1) {
        int first = -1, last = -1;
        int len = strlen(s);

        for(int i = 0; i < len; i++) {
            if(isVowel(s[i])) {
                if(first == -1)
                    first = i;
                last = i;
            }
        }

        if(first == -1)
            break;

        s[first] = toupper(s[first]);
        if(first != last)
            s[last] = toupper(s[last]);
    }

    printf("%s", s);
    return 0;
}

File: Daily Test/Java/Toggle Vowels - Both Ends 3.3.2026.java
import java.util.*;

public class Main {

    static boolean isVowel(char ch) {
        return ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u';
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();

        while(true) {
            int first = -1, last = -1;

            for(int i = 0; i < s.length(); i++) {
                if(isVowel(s.charAt(i))) {
                    if(first == -1)
                        first = i;
                    last = i;
                }
            }

            if(first == -1)
                break;

            char[] arr = s.toCharArray();
            arr[first] = Character.toUpperCase(arr[first]);

            if(first != last)
                arr[last] = Character.toUpperCase(arr[last]);

            s = new String(arr);
        }

        System.out.println(s);
        sc.close();
    }
}